streams/core.py

```diff
@@ -7,6 +7,7 @@
 from tornado.ioloop import IOLoop
 from tornado.queues import Queue
 from collections import Iterable
+from streams.graph import visualize
```

Comment:
one after the fact comment about this. I'd much rather not have the core import visualize, and also would be tempted to remove the member function `visualize`. I feel like it's too much overhead.
```python
from streams.graph import visualize
visualize(s1)
```
seems to me sufficient as opposed to `s1.visualize(....)`.

What do you think?